Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'active_model_serializers'
 gem 'business_time'
 gem 'holidays'
 gem 'factory_girl_rails'
+gem 'simple_form'
```

Comment:
We don't have an issues with simple_form in our projects, do we?

It's not the easiest to tweak the style of (as I found on when trying to make it more compatible with US Web Design Standards), but it does alleviate some of the tedium of form creation so that is good.